go/vt/vttablet/tabletmanager/vdiff/action_test.go

```diff
@@ -30,27 +34,64 @@ import (
 func TestPerformVDiffAction(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
 	defer cancel()
+	vdiffenv := newTestVDiffEnv(t)
+	defer vdiffenv.close()
+	keyspace := "ks"
+	workflow := "wf"
+	uuid := uuid.New().String()
 	tests := []struct {
-		name    string
-		vde     *Engine
-		req     *tabletmanagerdatapb.VDiffRequest
-		want    *tabletmanagerdatapb.VDiffResponse
-		wantErr error
+		name          string
+		vde           *Engine
+		req           *tabletmanagerdatapb.VDiffRequest
+		want          *tabletmanagerdatapb.VDiffResponse
+		expectQueries []string
+		wantErr       error
 	}{
 		{
 			name:    "engine not open",
 			vde:     &Engine{isOpen: false},
 			wantErr: vterrors.New(vtrpcpb.Code_UNAVAILABLE, "vdiff engine is closed"),
 		},
+		{
+			name: "delete by uuid",
+			req: &tabletmanagerdatapb.VDiffRequest{
+				Action:    string(DeleteAction),
+				ActionArg: uuid,
+			},
+			expectQueries: []string{
+				fmt.Sprintf(`delete from vd, vdt using _vt.vdiff as vd left join _vt.vdiff_table as vdt on (vd.id = vdt.vdiff_id)
```

Comment:
This is same as `sqlDeleteVDiffByUUID`

Reply:
Yeah, I intentionally did NOT use the variable as this way I can catch any accidental change to the query (in the variable).